src/content/docs/style-guide/word-choice/user-related-language-guidelines.mdx

```diff
@@ -61,9 +61,9 @@ Example use:
       </td>
 
       <td>
-Preferred: sentence case, bold. Alternatively, instead of bold, you can use italics or code-formatting: as long as it's clear that we're referring to something that has a definite, exact name. 
+Preferred for docs: sentence case, bold. Alternatively, for non-docs-site styling, you can use italics or code-formatting: as long as it's clear that we're referring to something that has a definite, exact name. 
```

Comment:
I don't think the colon after code-formatting: is grammatically correct. I think this should be a comma, or break into two sentences. "code-formatting. Whichever you choose, make sure we're referring to something..."